.github/workflows/pull-request.yml

```diff
@@ -1,46 +1,31 @@
 name: Pull Request
 
-on: [pull_request]
+on:
+  pull_request:
+    types: [opened, reopened, ready_for_review, synchronize]
 
 jobs:
   prchecks:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref }}
+          fetch-depth: 0
       - uses: actions/setup-python@v5
         with:
           python-version: "3.10"
-      - name: Configure pre-commit to skip branch name validation
-        run: |
-          echo "SKIP=branch-name-validation" >> $GITHUB_ENV
+      - name: Configure pre-commit to skip irrelevant checks
+        run: echo "SKIP=typescript-check-cli,typescript-check-api,typescript-check-ui" >> $GITHUB_ENV
       - uses: pre-commit/action@v3.0.1
   assignAuthor:
+    if: github.event.pull_request.user.type != 'Bot'
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Assign PR to Creator
-        run: |
-          if [ "$PR_AUTHOR_TYPE" != "Bot" ]
-            then
-              gh pr edit $PRNUM --add-assignee $PR_AUTHOR
-            fi
+        run: gh pr edit $PRNUM --add-assignee $PR_AUTHOR
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           PRNUM: ${{ github.event.pull_request.number }}
           PR_AUTHOR: ${{ github.event.pull_request.user.login }}
-          PR_AUTHOR_TYPE: ${{ github.event.pull_request.user.type }}
-
-  # Notify the integrations channel only when a Snyk auto merge fails pr checks
```

Comment:
Why are we removing notifications?

Reply:
It was only in core and I'm trying to get core to match all the other apps's pull-request where it makes sense.